sql_driver/lib.go

```diff
@@ -16,7 +19,17 @@ func (d *LibsqlDriver) Open(dbPath string) (driver.Conn, error) {
 	if strings.HasPrefix(dbPath, "file:") {
 		return (&sqlite3.SQLiteDriver{}).Open(dbPath)
 	}
-	return sqlddriver.SqldConnect(dbPath), nil
+	if strings.HasPrefix(dbPath, "wss://") {
+		jwt := os.Getenv("SQLD_AUTH_TOKEN")
```

Comment:
There is almost never a good reason for a library function to read its argument from an environment variable. We can pass the JWT in the URL as a query parameter (`ws://localhost?jwt=XXXX`) or as the userinfo (`ws://XXXX@localhost`).

Reply:
Alternatively, we can use a different method of instantiating the driver. The documentation for `sql.Open` says that "most users will open a database via a driver-specific connection helper function that returns a *DB.", or we can use `sql.OpenDB` with a `Connector`.

Reply:
done